### PR TITLE
Add Renesas Starter Kit Pro (Lava v2 only)

### PR DESCRIPTION
**Pull request by sjoerd-ccu**
Signed-off-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>

### CODE REVIEW

File: lib/device_map.py [hunk 1]
```diff
@@ -612,6 +612,17 @@
                'fastboot': False,
                'mach': 'renesas'}
 
+r8a7796_m3ulcb = {'device_type': 'r8a7796-m3ulcb',
+               'templates': [ 'generic-arm64-uboot-image-dtb-ramdisk-template.jinja2'],
+               'kernel_defconfig_blacklist': [],
+               'defconfig_blacklist': ['arm64-allnoconfig',
```
Comment (mattface) [inline lib/device_map.py:618]: We've found an issue with the blacklisting, so please update this to just be "allnoconfig" (drop the arch)

**Comment by mattface** [comment]
Please get the device type r8a7796-m3ulcb into upstream LAVA

**Comment by sjoerd-ccu** [comment]
Uhm, m3ulcb has been in upstream lava since march.. :)

**Comment by sjoerd-ccu** [comment]
Hrm, i see i was an idiot and submtted this config again in  PR #83, lets close this one (updated patch with your comments for balcklisting is in #83 now.

**Pull request closed without merging**